R/tweets_and_users.R

```diff
@@ -40,25 +40,13 @@ tweets_with_users <- function(x) {
 #' @rdname tweets_with_users
 #' @export
 users_with_tweets <- function(x) {
-  if (is.null(x) || (length(x) == 0L && !is.data.frame(x))) {
-    x <- data.frame()
-    attr(x, "tweets") <- data.frame()
-    return(x)
-  }
-  tweets <- status_object(x)
-  tweets <- tweets_df_(tweets)
-  users <- user_object(x)
-  users <- users_df_(users)
-  if (nrow(tweets) == nrow(users)) {
-    if (has_name_(users, "user_id")) {
-      tweets$user_id <- users$user_id
-      if (has_name_(users, "screen_name")) {
-        tweets$screen_name <- users$screen_name
-      }
-    }
-  }
-  attr(users, "tweets") <- tweets
-  join_rtweet(users)
+  users <- users_df_(user_object(x))
```

Comment:
Reading about the [information on user object](https://developer.twitter.com/en/docs/twitter-api/v1/accounts-and-users/follow-search-get-users/api-reference/get-users-lookup) it says:

>The order of user IDs or screen names may not match the order of users in the returned array. 

This is not part of this PR but couldn't find any confirmation/test to make sure that the information is correctly reported and the screen names and ID are reordered if needed. This is probably for a different PR/issue